crates/bevy_transform/src/lib.rs

```diff
@@ -101,19 +102,25 @@ impl Plugin for TransformPlugin {
             // add transform systems to startup so the first update is "correct"
             .add_startup_system_to_stage(
                 StartupStage::PostStartup,
-                systems::sync_simple_transforms.label(TransformSystem::TransformPropagate),
+                sync_simple_transforms
+                    .label(TransformSystem::TransformPropagate)
+                    // These systems cannot access the same entities,
+                    // due to subtle query filtering that is not yet correctly computed in the ambiguity detector
+                    .ambiguous_with(propagate_transforms),
```

Comment:
This should probably be written in the form of/include a `Todo` to remove this once fixed.